queries/r/textobjects.scm

```diff
@@ -79,44 +67,23 @@
   .
   (_) @parameter.inner
   .
-  ","? @_end)
+  (comma)? @_end)
   (#make-range! "parameter.outer" @parameter.inner @_end))
 
 ; number
 (float) @number.inner
 
 ; assignment
-(left_assignment
-  name: (_) @assignment.lhs
-  value: (_) @assignment.inner @assignment.rhs) @assignment.outer
-
-(left_assignment
-  name: (_) @assignment.inner)
+(binary_operator
+  lhs: (_) @assignment.inner @assignment.lhs
+  rhs: (_) @assignment.rhs) @assignment.outer
 
-(right_assignment
-  value: (_) @assignment.inner @assignment.lhs
-  name: (_) @assignment.rhs) @assignment.outer
-
-(right_assignment
-  name: (_) @assignment.inner)
-
-(equals_assignment
-  name: (_) @assignment.lhs
-  value: (_) @assignment.inner @assignment.rhs) @assignment.outer
-
-(equals_assignment
-  name: (_) @assignment.inner)
-
-(super_assignment
+((comma)
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(super_assignment
+((comma)
   name: (_) @assignment.inner)
```

Comment:
In the parser, comma never goes with a `name` field, only `parameter`. Please remove this as well